testsuite/smoke-test/src/storage.rs

```diff
@@ -43,6 +43,17 @@ async fn test_db_restore() {
     // set up: two accounts, a lot of money
     let mut account_0 = create_and_fund_account(&mut swarm, 1000000).await;
     let account_1 = create_and_fund_account(&mut swarm, 1000000).await;
+
+    // we need to wait for all nodes to see it, as client_1 is different node from the
+    // one creating accounts above
```

Comment:
I haven't checked myself.. but how does it make it flaky? A client can't transfer successfully if it doesn't see the correct balance?  

> client_1 is different node
you mean it's talking to a different node?

> from the one creating accounts above
irrelevant to this but I guess we need to make it more "user" friendly

Reply:
but yes, client (validator) cannot receive transaction request successfully, unless it sees that account exists. 

in the examples it failed, transfer below failed because account_0 didn't exist on client_1 validator yet